pilot/cmd/pilot-agent/app/cmd.go

```diff
@@ -311,6 +317,73 @@ func initProxy(args []string) (*model.Proxy, error) {
 	return proxy, nil
 }
 
+func applyExcludeInterfaces(ifaces []string) []string {
+	// Get list of excluded interfaces from pod annotation
+	annotations, err := bootstrap.ReadPodAnnotations("")
+	if err != nil {
+		log.Infof("Failed to read pod annotations to find exluded interfaces. Continuing without exclusions. Proxy IPAddresses: %v", ifaces)
```

Comment:
a bit worried about this. Will cause confusion for people IMO.

We already call this ReadAnnotations elsewhere (ConstructProxyConfig), so if its not there we would get 2 errors. But even in ConstructProxyConfig we only log "NotExist" at debug level.

This may seem minor - and it kind of is - but any "failed to ..." invariable causes support tickets etc even when they are harmless.

Reply:
Agreed. I will change the text.